cudax/include/cuda/experimental/__execution/concepts.cuh

```diff
@@ -148,6 +150,16 @@ _CCCL_CONCEPT dependent_sender = //
     requires(__is_dependent_sender<_Sndr>()) //
   );
 
+// Scheduler concepts:
+template <class _Sch>
+_CCCL_CONCEPT scheduler = //
+  _CCCL_REQUIRES_EXPR((_Sch), __declfn_t<_Sch> __sch) //
+  ( //
+    requires(__is_scheduler<_Sch>), //
+    schedule(__sch()), //
```

Comment:
Nitpick: I prefer to wrap expressions in requires clauses in `()`

A we often need this when commata are involved and also it discriminates from the `requires`

In this case I was wondering whether we had added another keyword to the concept handling macros